plugin/src/main/kotlin/com/github/triplet/gradle/play/ProcessPackageMetadataTask.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.triplet.gradle.play
+
+import com.android.build.gradle.api.ApkVariantOutput
+import com.github.triplet.gradle.play.internal.PlayPublishTaskBase
+import com.github.triplet.gradle.play.internal.ResolutionStrategy
+import org.gradle.api.tasks.TaskAction
+
+open class ProcessPackageMetadataTask : PlayPublishTaskBase() {
+    @TaskAction
+    fun processVersionCodes() = read { editId ->
+        val maxVersionCode = bundles().list(variant.applicationId, editId).execute().bundles
```

Comment:
Any reason why we are not using the tracks API here? We kind of imply that once users switched to bundles, they never go back to APKs (as we would resolve to the highest version code for bundles no matter if there's a higher apk version). 

Plus, I believe that the `bundles()/apks()` API would also list non-released versions from the artifact archive. Is that the desired behavior?

Reply:
You're right, I'm testing the tracks API right now and I'll push if it works.